packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -97,7 +99,7 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
   /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
   const holdButtonProps = usePropsFor(HoldButton);
 
-  /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(PSTN-calls) */
+  /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(one-to-n-calling) */
```

Comment:
Why are we adding two (one to n calling) here?

Reply:
good question, must be a result of my merging conflicts, will fix